src/index.js

```diff
@@ -0,0 +1,145 @@
+"use strict";
+
+const state = {
+    city: "Seattle",
+    tempCount: 0
+}
+
+let temperature = document.getElementById("temp-counter__span");
+
+// Function to set sky
+const skyImg = { 
+    "0" : "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️",
+    "1": "☁️ ☁️ ☁️ ☀️ ☁️ ☁️",
+    "2" : "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧",
+    "3" : "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨"
+};
+
+const changeSky = () => {
+    const inputSky = document.getElementById('skySelect').value;
+    const skyContainer = document.getElementById('garden-content__section');
+    let sky = '';
+    let skyColor = '';
+    if (inputSky === 'Cloudy') {
+      sky = skyImg[0];
+    //   skyColor = 'cloudy';
+    } else if (inputSky === 'Sunny') {
+        sky = skyImg[1];
+      skyColor = 'sunny';
+    } else if (inputSky === 'Rainy') {
+        sky = skyImg[2];
+      skyColor = 'rainy';
+    } else if (inputSky === 'Snowy') {
+        sky = skyImg[3];
+      skyColor = 'snowy';
+    }
+    skyContainer.textContent = sky;
+    // const gardenContent = document.getElementById('gardenContent');
+    // gardenContent.classList = `garden__content ${skyColor}`;
+  };
+
+// Function to set City and call getTemperature to change temp with city input
+
+    function saveCity() {
+        var userInput = document.getElementById("cityInput").value;
+        var displayElement = document.getElementById("displayCity");
+        displayElement.textContent = userInput;
+        // Call getTemperature function with user input and update temperature object
+        getTemperature(userInput).then((temp) => {
+            temperature.textContent = temp;
+            setColor(temp);
+        });
+    }
```

Comment:
This is valid JS but a more "modern" way to declare functions and variables would be to use the `const` + arrow function and using `const` instead of `var` here!